src/main/java/com/devcycle/sdk/server/local/DVCLocalClient.java

```diff
@@ -118,14 +119,15 @@ public <T> Variable<T> variable(User user, String key, T defaultValue) {
    * 
    * @param user (required)
    */
-  // TODO: Original return type should match the line below, uncomment once implemented and delete the void return
-  // public Map<String, Variable> allVariables(User user) throws DVCException {
-  public void allVariables(User user) {
+   public Map<String, Variable> allVariables(User user) throws JsonProcessingException {
     validateUser(user);
-
     addDefaults(user);
+    localBucketing.setPlatformData(user.getPlatformDataString());
```

Comment:
This should be handled via an override on `.toString()` - not via an explicit string call

Reply:
we still need a separate functions for just the platform data vs the whole user string, does it make sense to use toString for the whole user and then have a different function for getPlatformData?

Reply:
the idea I had was `user.getPlatformData().toString()`

Reply:
oh i see, good idea
Kaushal and I are discussing making PlatformData its own class since it won't change user to user, so I'll override toString on that class

Reply:
That's what I thought you were doing - as that's how we have it setup in every other platform. 
All good!